content/wasm-languages/zig.md

```diff
@@ -59,53 +133,83 @@ pub fn main() !void {
 }
 ```
 
-Now compile the program:
+Now compile and run the program using wasmtime:
 
 ```console
-zig build-exe -O ReleaseSmall -target wasm32-wasi src/main.zig
+$ zig build-exe -O ReleaseSmall -target wasm32-wasi src/main.zig
+$ wasmtime main.wasm    
+content-type: text/plain
+Hello, World!
 ```
 
-You can verify that it runs using `wasmtime`:
+Now you should have a `main.wasm` file:
 
 ```console
-$ wasmtime main.wasm
-content-type: text/plain
-
-Hello
+$ tree .
+.
+├── build.zig
+├── main.wasm
+├── main.wasm.o
+├── src
+│   └── main.zig
+└── zig-cache
```

Comment:
Not sure showing the contents of this is helpful - to me it felt scary rather than informative.